stockflux-securities-master/src/enums.js

```diff
@@ -0,0 +1,13 @@
+export const TableState = {
```

Comment:
One more suggestion here, but this is quite minor. It tends to be a convention that when you have unchanging, constant variables, particularly with enums that you define the variables in capital letters. So you would have something like:

    export const TableState = {
        LOADING: "loading",
        DELETING: "deleting",
        ERROR: "error",
        SUCCESS: "success"
    }

I think this would look and read better. Same goes for the `InputFormState` below too.

Reply:
DONE